src/EA.Weee.Web/Areas/Producer/Mappings/ToRequest/ToAddSignatoryAndCompleteRequestCreator.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace EA.Weee.Web.Areas.Producer.Mappings.ToRequest
+{
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Core.Organisations;
+    using EA.Weee.Requests.Organisations.DirectRegistrant;
+    using EA.Weee.Web.Areas.Producer.ViewModels;
+    using EA.Weee.Web.Requests.Base;
+
+    public class ToAddSignatoryAndCompleteRequestCreator : IRequestCreator<AppropriateSignatoryViewModel, AddSignatoryAndCompleteRequest>
+    {
+        private readonly IMapper mapper;
+
+        public ToAddSignatoryAndCompleteRequestCreator(IMapper mapper)
+        {
+            this.mapper = mapper;
```

Comment:
@stevelee-civica think we need unit tests for this one, sorry if I missed it

Reply:
added unit tests